### PR TITLE
Automated cherry pick of #3943: add release ci

### PR DESCRIPTION
**Pull request by gy95**
Cherry pick of #3943 on release-1.11.

#3943: add release ci

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        target:
+          - cloudcore
+          - admission
+          #- edgecore
+          - edgesite-agent
+          - edgesite-server
+          - csidriver
+          - iptablesmanager
+          - edgemark
+          - installation-package
```
Comment (gy95) [inline .github/workflows/release.yml:62]: /hold
waiting for add controllermanager

Comment (gy95) [reply .github/workflows/release.yml:61]: /hold-cancel

Comment (gy95) [reply .github/workflows/release.yml:61]: /hold remove
/remove hold

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**